### DISCUSSION TITLE
Add more than one relation at a time for relational fields

### DISCUSSION THREAD

**Discussion by dlindberg** [craftcms/cms#15127] [Ideas] [answered]
On relational fields, (for entries, assets, categories, etc.) the item picker that comes up will let users check off several things, but only the first one will actually be added as a relationship. For fields where multiple relationships can be set, it would be a nice quality of life improvement if all of the ones checked off by the user got added (so long as it doesn't exceed the maximum number of relations for the field).

**Comment by thupsi** [comment, marked as answer]
This is already the case. To make sure I just tested on 5.1.8 and it works as intended. Maybe there's some detail in your field config that causes it to only add the first item?

**Reply by dlindberg** [reply]
Thanks, looks like this might actually be an issue with structures in structure view. Just confirmed works fine when viewing the list as a list, but when viewing as a structure it will only add one tree of it at a time. In our case I can force a filter on users to be able to just pick one level of things.
